internal/cli/atlas/backup/schedule/delete.go

```diff
@@ -61,8 +61,8 @@ func DeleteBuilder() *cobra.Command {
 		Annotations: map[string]string{
 			"clusterNameDesc": "Name of the cluster.",
 		},
-		Example: fmt.Sprintf(`  The following example deletes all backup schedules of the cluster Cluster0:
-  $ %s backup schedule delete Cluster0`, cli.ExampleAtlasEntryPoint()),
+		Example: fmt.Sprintf(`  # The following example deletes all backup schedules of the cluster Cluster0:
+  %s backup schedule delete Cluster0`, cli.ExampleAtlasEntryPoint()),
```

Comment:
[q] Are we adding `#` only when the command has more than 1 example or all the time?

Reply:
`#` is a comment marker, it's been added to anything that's not "runnable"

Reply:
As Gus said, I added just on things that have an introductory sentence (regardless of number of examples). The idea was to make it so that folks can copy the code block in the docs and run it without the intro text or the `$` making the code invalid.